.gitignore

```diff
@@ -1,17 +1,51 @@
-/build
-library/build
-demo/build
```

Comment:
would you please keep the library/build and demo/build exclusions to be safe, even though the current files are accounted for wit the .class, bin/, gen/ .apk, etc.